chart/openfaas/templates/crd.yaml

```diff
@@ -65,5 +65,27 @@ spec:
                 memory:
                   type: string
                   pattern: "^[0-9]+(Mi|Gi)"
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: policies.openfaas.com
+spec:
+  group: openfaas.com
+  version: v1
+  versions:
+    - name: v1
+      served: true
+      storage: true
+    - name: v1alpha1
+      served: true
+      storage: false
+  names:
+    plural: policies
+    singular: policy
+    kind: Policy
+    shortNames:
+    - pol
+  scope: Namespaced
```

Comment:
What's the strategy for copying/patching the generated schema into your CRD templates for the Helm chart?

For reference, we achieve this in cert-manager by adding in Helm templating directives dynamically (i.e. at the time the Helm chart is built), by appending and prepending the required `{{ if .. }}` and `{{ end }}` blocks.

Reply:
I was going to copy and paste it into the middle of the helm chart file after each change -> https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/templates/crd.yaml

We could update the bash script to populate this file with a prefix/suffix and the generated content in the middle.